catroid/src/main/java/org/catrobat/catroid/io/ProjectAndSceneScreenshotLoader.java

```diff
@@ -115,6 +120,41 @@ public File getScreenshotFile(String projectName, String sceneName, boolean isBa
 		return screenshotLoader.getScreenshotFile();
 	}
 
+	public String getScreenshotScene(File projectDir) {
+		FilenameFilter filter = new FilenameFilter() {
+
+			public boolean accept(File f, String name) {
+				return name.endsWith("screenshot.png");
+			}
+		};
+		File[] projectScreenshots = projectDir.listFiles(filter);
+		if (projectScreenshots == null || projectScreenshots.length != 0) {
+			return null;
+		}
+		List<File> screenshots = new ArrayList<>();
+		for (File scene : projectDir.listFiles()) {
+			File[] sceneScreenshots = scene.listFiles(filter);
+			if (sceneScreenshots != null && sceneScreenshots.length > 0) {
+				screenshots.addAll(Arrays.asList(scene.listFiles(filter)));
+			}
+		}
+		if (screenshots.size() < 1) {
+			return null;
+		}
+		Collections.sort(screenshots, new Comparator<File>() {
+			@Override
+			public int compare(File screenshot2, File screenshot1) {
+				return Long.compare(screenshot1.lastModified(), screenshot2.lastModified());
+			}
+		});
+		for (File screenshot : screenshots) {
+			if (screenshot.getName().equals("manual_screenshot.png")) {
+				return screenshot.getParentFile().getName();
+			}
+		}
+		return screenshots.get(0).getParentFile().getName();
```

Comment:
same as line 152, `getParentFile()` returns _abstract pathname of the parent directory named by this abstract pathname, or null if this pathname does not name a parent_ as mentioned in https://developer.android.com/reference/java/io/File#getParentFile() 
I think you need the name of the file, right?

Reply:
I need the name of the parent file (the scene where the screenshot is saved in), that's why I take the parent and then the name. How would you do it?